app/src/main/java/com/framgia/soundclound/screen/playtrack/PlayTrackActivity.java

```diff
@@ -3,22 +3,27 @@
 import android.content.Context;
 import android.content.Intent;
 import android.databinding.DataBindingUtil;
+import android.support.annotation.Nullable;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 
 import com.framgia.soundclound.R;
-import com.framgia.soundclound.data.source.local.SharePreferences;
+import com.framgia.soundclound.data.model.Track;
+import com.framgia.soundclound.data.source.TrackRepository;
+import com.framgia.soundclound.data.source.local.realm.TrackLocalDataSourceimpl;
 import com.framgia.soundclound.databinding.ActivityPlayTrackBinding;
 
+import java.util.List;
 
 /**
  * Created by Sony on 1/11/2018.
  */
 public class PlayTrackActivity extends AppCompatActivity {
+    private static final String TAG = PlayTrackActivity.class.toString();
+    private TrackRepository mTrackRepository;
 
-
-
-    public static Intent getInstance(Context context) {
+    public static Intent getInstance(Context context, @Nullable List<Track> trackList, @Nullable int position) {
+        Intent intent = new Intent(context, PlayTrackActivity.class);
```

Comment:
ERROR: [STYLE][DLS_DEAD_LOCAL_STORE] Dead store to intent in com.framgia.soundclound.screen.playtrack.PlayTrackActivity.getInstance(Context, List, int)
http://findbugs.sourceforge.net/bugDescriptions.html#DLS_DEAD_LOCAL_STORE